src/librustdoc/html/static/main.js

```diff
@@ -380,8 +380,22 @@ function getSearchElement() {
                 set_fragment(cur_id);
             }
         } else if (hasClass(document.getElementById("help"), "hidden") === false) {
-            addClass(document.getElementById("help"), "hidden");
-            removeClass(document.body, "blur");
+            var x = ev.target;
+            var is_inside_help_popup = false;
+            do {
+                x = findParentElement(x, "DIV");
+                if (x) {
+                    x = x.parentNode;
+                }
+                if (x && x.tagName === "ASIDE") {
+                    is_inside_help_popup = true;
+                    break;
+                }
+            } while(x);
```

Comment:
Could this entire loop be replaced with:

```javascript
            var is_inside_help_popup = document.getElementById("help").contains(ev.target);
```

Reply:
@GuillaumeGomez after making this change, ping me

Reply:
@kinnison Awesome! :o

Reply:
I'd not, I hadn't *tested* that, but I assume @GuillaumeGomez has.

Reply:
Actually, I'm wondering about something, let me check it further!